### ISSUE TITLE
Separate Nextcloud News and community blogs

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/nextcloud.com#404]
Currently when you go to "Blog" you get to the planet and are overwhelmed by community posts which tend to be not that relevant.

We should have two entries in the navigation dropdown:
- Articles (only having news from Nextcloud)
- Community (the current planet)

Cc @nextcloud/website

**Comment by MariusBluem** [comment]
What about showing Nextcloud News there where the current Planet is - and the community content as a sidebar with the headlines, profile-picture and a link to their external blog 🤔

**Comment by Espina2** [comment]
I think we should move the blog and the news to the ghost platform https://ghost.org/

**Comment by jancborchardt** [comment]
@MariusBluem yes, good call! :) What do you think @jospoortvliet?

@Espina2 this is an entirely different topic? (Also, why should we move if we already have it here integrated with Wordpress?)

**Comment by Espina2** [comment]
Because we have problems for example with the RSS feed. It is not well integrated and that is the principal reason that I think we should move.

**Comment by jospoortvliet** [comment]
I can't say exactly what direction we should take technically, I can only tell you I have no clue whatsoever how to make the change Jan/Marius proposed. And yes, I agree with @MariusBluem - that would be great. But how???

**Comment by MariusBluem** [comment]
First step could be to create a 2nd Blog roll and remove every Collaborator excecpt "Nextcloud" from the First, and do all Community collaborators in the 2nd 😁😅

**Comment by jospoortvliet** [comment]
Agreed. But with things handled via Contribook I don't know how to do that :(

**Comment by mathiasconradt** [comment]
I like @Espina2 's idea of moving the blog out, no hassle with managing code, css. Medium (https://medium.com/) would also be an alternative, and it's quite popular, already has a user base that reads the articles, since it automatically gets recommended / into their global feed.

**Comment by MariusBluem** [comment]
> I like @Espina2 's idea of moving the blog out, no hassle with managing code, css. Medium (https://medium.com/) would also be an alternative, and it's quite popular, already has a user base that reads the articles, since it automatically gets recommended / into their global feed.

I like it :+1: Would cause less bugs in our blogging infrastructure and make it 100% responsive with less work :)

**Comment by jancborchardt** [comment]
Ehm … Medium is one of the very things we are aiming to make obsolete. It's a completely centralized publishing platform. Why would we use that?

I also don't see how switching to Ghost is worth the effort if instead we should debug the original RSS issues. They might stem from a plugin or other misconfiguration.

Otherwise we just introduce yet another tool in our workflow, and that is going to make it harder in the end, not easier.

**Comment by mathiasconradt** [comment]
> Medium is one of the very things we are aiming to make obsolete. It's a completely centralized publishing platform.

You mean in general, or for our internal use? Why would we want to make it obsolete? It's good and useful. Just because something is centralized doesn't make it bad by default, especially not for platforms that allow to push public news out. I mean, so is Twitter, etc. and we still use it. openpr.de is also similar, and it's still a good and useful platform. The same goes for LinkedIn Pulse.
It brings readers, who didn't search for Nextcloud explicitly, to the blog via its recommendation system (similar posts, top stories for you, technology channel).

> Why would we use that?

Medium has a good outreach and features and brings visitors to the blog via its recommendation system (similar posts, top stories for you, technology channel), it's also very popular. Medium is a SaaS and we don't have fiddle around with css and code. But most importantly it has a good outreach, plus it's good (or at least not worse) for SEO I think.

**Comment by jancborchardt** [comment]
Sure we could crosspost there – just like we do on Facebook and Twitter. But since we have a Wordpress setup already, we shouldn't move the main source over.
